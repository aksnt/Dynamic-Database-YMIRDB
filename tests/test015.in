SNAPSHOT
SNAPSHOT
SNAPSHOT
set a 1 2 3
SNAPSHOT
get a
gEt a
sEt b 3 4 a
cHeCkoUt 3
RollBAcK 1
get A
get a
bye
