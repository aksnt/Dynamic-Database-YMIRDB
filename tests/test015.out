> saved as snapshot 1

> saved as snapshot 2

> saved as snapshot 3

> ok

> saved as snapshot 4

> [1 2 3]

> [1 2 3]

> ok

> ok

> ok

> no such key

> no such key

> bye
